test/functional/wallet_bumpfee.py

```diff
@@ -100,28 +99,55 @@ def run_test(self):
         test_small_output_with_feerate_succeeds(self, rbf_node, dest_address)
         test_no_more_inputs_fails(self, rbf_node, dest_address)
 
-    def test_invalid_parameters(self, node, dest_address):
-        txid = spend_one_input(node, dest_address)
-        # invalid estimate mode
-        assert_raises_rpc_error(-8, "Invalid estimate_mode parameter", node.bumpfee, txid, {
-            "estimate_mode": "moo",
-        })
-        assert_raises_rpc_error(-3, "Expected type string", node.bumpfee, txid, {
-            "estimate_mode": 38,
-        })
-        assert_raises_rpc_error(-3, "Expected type string", node.bumpfee, txid, {
-            "estimate_mode": {
-                "foo": "bar",
-            },
-        })
-        assert_raises_rpc_error(-8, "Invalid estimate_mode parameter", node.bumpfee, txid, {
-            "estimate_mode": Decimal("3.141592"),
-        })
-        # confTarget and conf_target
-        assert_raises_rpc_error(-8, "confTarget and conf_target options should not both be set", node.bumpfee, txid, {
-            "confTarget": 123,
-            "conf_target": 456,
-        })
+    def test_invalid_parameters(self, rbf_node, peer_node, dest_address):
```

Comment:
This is a bit hard to review. My approach would currently be to check line by line whether all test cases are accounted for but that would probably be easier if the reordering of code and the refactoring to traverse the test cases had been separated into two commits. What would a reviewer usually be expected to do here?

Alternatively, I suspect that there might be better diff tools that can follow a change like this. :thinking:

Reply:
Sorry about that. Yes, there are a few options. In this case, a helpful one might be

`git show 6fa5686 -w --color-moved=dimmed-zebra`

For other cases, maybe git show or git diff with `--word-diff`.

Or to ignore all added lines having a matching identical removed line (empty result proves move-only):

`git diff 6fa5686f~1..6fa5686f | grep -E '^[+-][^+-]' | cut -b2- | sort | uniq -c | grep -v '^ *2'`

People like practicalswift, promag, MarcoFalke, jnewbery, and harding likely know some more good ones.